### PR TITLE
Effects: Mark transfer-effect deprecated, document .transfer()

### PR DESCRIPTION
**Pull request by jzaefferer**
Ref gh-242

First commit to document the effects rewrite, more to come.

### CODE REVIEW

File: entries/transfer.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	</longdesc>
+	<signature>
+		<argument name="options" type="Object">
+			<property name="to" type="Selector">
+				<desc>The target of the transfer effect.</desc>
+			</property>
+			<property name="className" type="String" default="null">
+				<desc>A class to add to the transfer element, in addition to <code>ui-effects-transfer</code>.</desc>
+			</property>
+			<property name="duration" type="Number" default="400">
```
Comment (scottgonzalez) [inline entries/transfer.xml]: Use the content from https://github.com/jquery/api.jqueryui.com/blob/master/includes/animation-argument-duration.xml.

File: entries/transfer.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+				<desc>How long the animation runs.</desc>
+			</property>
+			<property name="easing" type="String" default="swing">
```
Comment (scottgonzalez) [inline entries/transfer.xml]: Use the content from https://github.com/jquery/api.jqueryui.com/blob/master/includes/animation-argument-easing.xml.

File: entries/transfer.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+				<desc>The easing to use for the animation.</desc>
+			</property>
+		</argument>
+		<argument name="done" type="Callback">
```
Comment (scottgonzalez) [inline entries/transfer.xml]: Rename to `complete` for consistency with other animation methods. Use the content from https://github.com/jquery/api.jqueryui.com/blob/master/includes/animation-argument-complete.xml.

Comment (scottgonzalez) [inline entries/transfer.xml]: Actually, since we're in a new argument here, we can use that include :-)

**Comment by scottgonzalez** [comment]
Some minor changes, but it looks good. The `.transfer()` method itself has a bug though :-/ http://bugs.jqueryui.com/ticket/12223 I'll work on that today.

**Comment by jzaefferer** [comment]
Updated

File: entries/transfer.xml (hunk 1 above)
Comment (scottgonzalez) [inline entries/transfer.xml]: You only got part of this; there are two types. It's also optional, not sure if we support that for properties...

**Comment by jzaefferer** [comment]
Merged 2af9f44 into `1-12` branch

**Pull request closed without merging**